lib/utils/vpc-utils.ts

```diff
@@ -0,0 +1,25 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import { Annotations, Token, Tags } from '@aws-cdk/core';
+
+/**
+ * Tags Subnets within a VPC
+ * @param scope
+ * @param type
+ * @param subnets
+ * @param tag
+ * @param value
+ */
+export function tagSubnets(scope: cdk.Construct, subnets: ec2.ISubnet[], tag: string, value: string): void {
```

Comment:
why do we need it and where is it used?

Reply:
Obsolete based on your latest comments. Will be addressing it there.